python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -370,13 +374,12 @@ class RandomForestClassifier(Base):
 
     def _get_model_info(self):
         cdef ModelHandle cuml_model_ptr = NULL
-        task_category = 1
+        task_category = CLASSIFICATION_CATEGORY
```

Comment:
once the multi class fil is added to cuml then this will be updated to pass the num_classes, `task_category=num_classes` as task category, i can change it for now if you prefer.